libraries/ZigBeeNet/Database/IZigBeeNetworkDataStore.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using ZigBeeNet;
+
+namespace ZigBeeNet.Database
```

Comment:
I would agree to @nicolaiw because database mostly seems to be a database like sql, but your idea is also able to store it everywhere (files, etc)

Reply:
I agree that DataStore would be a better name. However the java project use Database and I think at this point it's a good idea to keep our code as close as possible from the java version because it is very likely that we are going to port a lot of other changes from this project. If you strongly feel otherwise I can change it, just let me know.

Reply:
Yeah, that make sense 😁